.github/ISSUE_TEMPLATE/feature_request.yml

```diff
@@ -0,0 +1,37 @@
+name: "💡 Feature Request"
+description: If you have a suggestion for how to improve Everest
+labels: ["enhancement"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to suggest a new feature! Please fill out this form as completely as possible.
+
+        Please search to see if an issue already exists for the feature you are suggesting.
+
+  - type: textarea
+    attributes:
+      label: What problem are you trying to solve?
+      description: A concise description of what the problem is.
+      placeholder: I have an issue when [...]
+    validations:
+      required: true
+
+  - type: textarea
+    attributes:
+      label: How should the problem be solved?
+      description: What do you want to happen? Add any considered drawbacks.
+    validations:
+      required: true
+
+  - type: textarea
+    attributes:
+      label: Any alternatives you've considered?
+      description: Is there another way to solve this problem that isn't as good a solution?
+    validations:
+      required: true
```

Comment:
I am doubt that someone will fill this

Reply:
yeah, that's why it is not a required field, so they can skip it 🤷🏻

Reply:
yeah didn't realize it was not optional.. fixed